CONTRIBUTING.md

```diff
@@ -1,3 +1,35 @@
+* [Welcome!](#welcome)
```

Comment:
I would add a `Table of Contents` header above this line, just to make it clear and consistent.

Reply:
Should the table of content list itself?

Reply:
I don't think that's necessary.